docs/user_guide/write_api/push_job.md

```diff
@@ -9,10 +9,11 @@ permalink: /docs/user_guide/write_api/push_job
 The Push Job takes data from a Hadoop grid and writes it to Venice. 
 
 ## Use Cases
-There are two modes the Push Job can run in:
+There are three modes the Push Job can run in:
 
 - Full Push (default)
 - Incremental Push
+- Targeted Region Push
```

Comment:
Hmm... I think it's confusing to put this here. Targeted pushes are an option on Full Pushes, but in theory they could apply to inc pushes as well (with more changes, which we're not working on ATM). So IMHO it is orthogonal to inc/full push, rather than a third mode.

Reply:
I agree this may be confusing if we consider it as 3rd mode. I'd like to still introduce this in this `push_job.md` though since we don't need to create a new file for each new feature. Let me move this to a new section.